core/serialisation/src/main/java/uk/gov/gchq/gaffer/serialisation/implementation/Ordered/OrderedRawDateSerialiser.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.serialisation.implementation.Ordered;
+
+import uk.gov.gchq.gaffer.exception.SerialisationException;
+import uk.gov.gchq.gaffer.serialisation.ToBytesSerialiser;
+import java.util.Date;
+
+public class OrderedRawDateSerialiser implements ToBytesSerialiser<Date> {
```

Comment:
As we are adding a new serialiser for the Java Date object, would there be any benefit to adding a serialiser for the Java 8 DateTime family of objects? It's heavily discouraged to use the old Java Date classes in favour of the new APIs (e.g. http://www.oracle.com/technetwork/articles/java/jf14-date-time-2125367.html) - should we be supporting these?